Add php-cs-fixer

lukasoppermann opened the pull request:
This is the beginning of adding a style fixer, which would be good to fix #63

jrean commented:
Hi, I really need to dig into php cs fixer before merging. I never used it (yet)

lukasoppermann commented:
sure, its very easy. As you can see I added 2 composer scripts, `test` just does a try run, meaning nothing is changed and it just shows the errors, `fix` actually fixes the errors.

To prepare for StyleCI I added `sllh/php-cs-fixer-styleci-bridge` which basically matches your fixers with the ones from your styleCI config `.styleci.yml`. I just set it to recommended, for now, as this is a pretty good setting in my opinion.

However this PR obviously only effects development, and has no impact on the package from a users point of view.

lukasoppermann commented:
@jrean can you pull this too?

jrean commented:
Yep. Can you please rebase?
And before I need to do homework... Create accounts and Co.

lukasoppermann commented:
Sure, I can rebase. However you do not need to create  Accounts to pull
this. It will be useful even before you create any accounts.

On Sun, Sep 25, 2016, 5:11 PM Jean Ragouin <notifications@github.com> wrote:

> Yep. Can you please rebase?
> And before I need to do homework... Create accounts and Co.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jrean/laravel-user-verification/pull/64#issuecomment-249427109>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAxquniLpbfvNF6ZKNpSViiIm9-e-8FYks5qto82gaJpZM4J8fXv>
> .
>
-- 
—
*veare*
Lukas Oppermann

M lukas@vea.re
W www.vea.re
P +49 163 162 69 47

jrean commented:
Ok. So I will pull :-)

lukasoppermann commented:
@jrean ready to be merged.

jrean commented:
Gonna merge it. Don't have time now to handle travis and style accounts. Tomorrow I guess.

jrean commented:
Thank you
